src/index.js

```diff
@@ -1 +1,61 @@
 import './index.css';
+
+// ------------ API ------------ //
+const gameId = process.env.GAME_ID;
+const baseUrl = process.env.BASE_URL;
+
+const getScores = async () => {
+  try {
+    const response = await fetch(`${baseUrl}/games/${gameId}/scores/`);
+    const data = await response.json();
+    return data.result;
+  } catch (e) {
+    return `Error: ${e}`;
+  }
+};
+
+const addScore = async (user, score) => {
+  try {
+    const request = await fetch(`${baseUrl}/games/${gameId}/scores/`, {
+      method: 'POST',
+      body: JSON.stringify({
+        user,
+        score,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    });
+    const response = await request.json();
+
+    return response.result;
+  } catch (e) {
+    return e;
+  }
+};
+
+const socresList = document.querySelector('.scores');
+const displayScores = async () => {
+  const scores = await getScores();
+  socresList.innerHTML = '';
+  scores.forEach((score) => {
+    socresList.innerHTML += `
+      <li>
+        <p>${score.user}: ${score.score}</p>
+      </li>
+    `;
+  });
+};
```

Comment:
- [ ] When I submit the name and score to score list, I didn't get score and name on score list and  it's throw an error  on console with an error "Uncaught (in promise) TypeError: scores.forEach is not a function" as show in the screenshot below. Mostly The "forEach is not a function" error occurs when we call the `forEach()` method on a value that is not of type array, Map , or Set . To solve the error, make sure to only call the `forEach` method on arrays, Map or Set objects. Kindly Fix it.

![Capture](https://user-images.githubusercontent.com/93598189/202275099-f2a47d87-fc70-43cd-9186-2121bbac95e5.PNG)

Reply:
@Abenezer-Tilahun it's working on my laptop no errors

![Screenshot from 2022-11-16 21-45-45](https://user-images.githubusercontent.com/32738452/202279354-6f9ae15a-cf4c-42e2-9484-b0db79c36b5a.png)


![Screenshot from 2022-11-16 21-46-01](https://user-images.githubusercontent.com/32738452/202279374-f3732bdb-7d5c-4dae-84cf-a24c3d8e508d.png)

Reply:
Hello @belalgamal46 Okay I see that , You can submit for review and it will be approved.

Reply:
@Abenezer-Tilahun It was because I have a .env file I will implement it normal javascript

Reply:
@Abenezer-Tilahun thanks 😊😊